mu-plugins/central-hub/tests/phpunit/unit/config-store/_theStore.php

```diff
@@ -181,6 +182,62 @@ public function test_should_overwrite_a_stored_config_using_same_key() {
 		$this->assertSame( $new_config, getConfig( __METHOD__ ) );
 
 		// Clean up.
-		_the_store( __METHOD__, null, true );
+		self::empty_the_store_by_keys( [ __METHOD__] );
+	}
+
+	/******************************************************************************
+	 * Helper functions to clean and reset `_the_store` before and after each test.
+	 ******************************************************************************/
+
+	/**
+	 * Empty all of the configs from the store.
+	 *
+	 * @param array $configs Optional. Array of configs stored in the store.
+	 *
+	 * @return void
+	 * @throws \Exception
+	 * @since 1.0.0
+	 *
+	 */
+	protected static function empty_the_store( $configs = [] ) {
```

Comment:
@rgadon107 The knowledge of how to empty the store exists in this internal function; therefore, it exists in this test class itself. There's no need to carry this extra helper methods.